src/main/java/com/backbase/ct/bbfuel/data/PocketsDataGenerator.java

```diff
@@ -0,0 +1,39 @@
+package com.backbase.ct.bbfuel.data;
+
+import static com.backbase.ct.bbfuel.data.CommonConstants.POCKETS_CURRENCY;
+
+import com.backbase.ct.bbfuel.util.CommonHelpers;
+import com.backbase.ct.bbfuel.util.GlobalProperties;
+import com.backbase.dbs.pocket.tailor.client.v1.model.Currency;
+import com.backbase.dbs.pocket.tailor.client.v1.model.PocketGoalRequest;
+import com.backbase.dbs.pocket.tailor.client.v1.model.PocketPostRequest;
+import com.github.javafaker.Faker;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+public class PocketsDataGenerator {
+
+    private static final Faker faker = new Faker();
+    private static final GlobalProperties GLOBAL_PROPERTIES = GlobalProperties.getInstance();
+
+    private PocketsDataGenerator() {
+        throw new IllegalStateException("DataGenerator class");
+    }
+
+    /**
+     * Generate PocketPostRequest.
+     * @return PocketPostRequest
+     */
+    public static PocketPostRequest generatePocketPostRequest() {
+
+        BigDecimal amount = CommonHelpers.generateRandomAmountInRange(100L, 9999L);
+        String currency = GLOBAL_PROPERTIES.getString(POCKETS_CURRENCY);
+
+        return new PocketPostRequest()
+            .name(faker.lorem().characters(80))
```

Comment:
80 characters for name and icon seem excessive:
`ksjlakjsdlkajlkcn salkdjalksdjlaksjd  laksjdalksjd lkasjd  lkasawidjowidjoqwidj`

Btw, we could use city names and other fake data instead of `lorem`, that will make more sense, have a look at the available options: https://github.com/DiUS/java-faker#fakers

Reply:
I will use some fancy fake data that makes more sense